### ISSUE TITLE
Convert to markdown on paste

### ISSUE THREAD

**Issue by Amparose** [careteditor/caret#600]
As Caret is the only markdown editor (that I know of) that allows you to paste a in an image from clipboard, I am disappointed to see that it does not convert copied rich text (e.g. headings, text, and bullet points from a webpage) to markdown on paste.

I'd like to request this feature.

**Comment by borekb** [comment]
Good idea, I currently use http://euangoddard.github.io/clipboard2markdown/.
